www/docs/en/dev/guide/platforms/android/plugin.md

```diff
@@ -155,27 +158,24 @@ thread, but do not want to block the `WebCore` thread either:
 
 ## Adding Dependency Libraries
 
-If a plugin requires additional libraries to work, you can use
-one of the following approaches to add them via `config.xml`.
+If your Android plugin has extra dependencies, they must be listed in the
+`plugin.xml` in one of two ways.
 
-Option A. Via _Gradle_ reference, for example:
-
-        <framework src="com.android.support:support-v4:+" />
-
-This is a recommended approach as it allows multiple plugins
-to refer to the same dependency library such as _gson_,
-_android-support-v4_, _google-play-services_, etc and
-_Gradle_ will resolve duplicate dependencies using its
+The preferred way is to use the `<framework />` tag (see the
+[Plugin Specification](../../../plugin_ref/spec.html) for more details).
+Specifying libraries in this manner allows them to be resolved via Gradle's
 [Dependency Management logic](https://docs.gradle.org/current/userguide/dependency_management.html).
-
-Option B. As _JAR_ files placed to some plugin's folder and
-linked using `lib-file`, for example:
-
-        <lib-file src="src/android/libs/gcm.jar"/>
-
-We recommend using this approach only if you are sure that
-dependency jar is plugin specific and won't be used by
-other plugins. Otherwise, there will be platform build issue.
+This allows commonly used libraries such as _gson_, _android-support-v4_,
+and _google-play-services_ to be used by multiple plugins without conflict.
+
+The second option is to use the `<lib-file />` tag to specify the location of
```

Comment:
Does `framework` support a jar reference too? Are these mutually exclusive or `framework` is always preferred over `lib-file`?